### PR TITLE
docs: update README

### PR DESCRIPTION
**Pull request by rev-chaos**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Which docs this PR relation**:

Ref #

**Which toolchain this PR adaption**:

No Breaking Change

**Special notes for your reviewer**:

### CI Switch

ci-runs-only: [Chaos CI, Cargo Clippy, Coverage Test, E2E Tests, Code Format, Unit Tests]

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,28 @@
 # Axon
 
+> Axon is still under active development and considered to be a work in progress.
+
+## About Axon
+
+Axon is a sidechain framework based on [Muta](https://github.com/nervosnetwork/muta) with UDT token staking on CKB.
+Dapp developers could use Axon toolkit to create and distribute chain-specific UDT on CKB, chain token holders could staking UDT to become an Axon chain validator, or just deposit it into Axon chain for use in dapps.
```
Comment (xying21) [inline README.md]: Dapp developers can use Axon toolkit to create and distribute chain-specific UDT on CKB. Chain token holders can stake UDT to become an Axon chain validator or just deposit UDT into Axon chain for use in Dapps.

File: README.md [hunk 2]
```diff
@@ -1,5 +1,28 @@
 # Axon
 
+> Axon is still under active development and considered to be a work in progress.
```
Comment (xying21) [inline README.md]: Axon is still in active development and a work in progress.

File: README.md [hunk 3]
```diff
@@ -1,5 +1,28 @@
...
+
+## About Axon
+
+Axon is a sidechain framework based on [Muta](https://github.com/nervosnetwork/muta) with UDT token staking on CKB.
+Dapp developers could use Axon toolkit to create and distribute chain-specific UDT on CKB, chain token holders could staking UDT to become an Axon chain validator, or just deposit it into Axon chain for use in dapps.
+Axon will support [interoperability 2.0](https://medium.com/nervosnetwork/blockchain-abstraction-and-interoperability-2-0-eea98d81b7b6) and be 100% Ethereum compatible, like [Godwoken](https://github.com/nervosnetwork/godwoken).
+
+As one of the layer-2 solutions of [Nervos](https://www.nervos.org/), Axon could achieve thousands of TPS with the help of the high-performance consensus, [Overlord](https://github.com/nervosnetwork/overlord).
```
Comment (xying21) [inline README.md]: As a layer-2 solution on [Nervos](https://www.nervos.org/), Axon can achieve thousands of TPS with the help of [Overlord](https://github.com/nervosnetwork/overlord), a high-performance consensus.

File: README.md [hunk 4]
```diff
@@ -1,5 +1,28 @@
...
+> Axon is still under active development and considered to be a work in progress.
+
+## About Axon
+
+Axon is a sidechain framework based on [Muta](https://github.com/nervosnetwork/muta) with UDT token staking on CKB.
+Dapp developers could use Axon toolkit to create and distribute chain-specific UDT on CKB, chain token holders could staking UDT to become an Axon chain validator, or just deposit it into Axon chain for use in dapps.
+Axon will support [interoperability 2.0](https://medium.com/nervosnetwork/blockchain-abstraction-and-interoperability-2-0-eea98d81b7b6) and be 100% Ethereum compatible, like [Godwoken](https://github.com/nervosnetwork/godwoken).
+
+As one of the layer-2 solutions of [Nervos](https://www.nervos.org/), Axon could achieve thousands of TPS with the help of the high-performance consensus, [Overlord](https://github.com/nervosnetwork/overlord). 
+Thus just provides Nervos DApp developers with one more option. For low throughput high-value scenarios such as DeFi where security is usually a higher concern than speed, Godwoken is a perfect choice.
```
Comment (xying21) [inline README.md]: This offers Nervos DApp developers one more option. Godwoken is perfect for scenarios that involve low throughput and high value, such as DeFi, where security is usually more important than speed.

File: README.md [hunk 5]
```diff
@@ -1,5 +1,28 @@
...
+
+## About Axon
+
+Axon is a sidechain framework based on [Muta](https://github.com/nervosnetwork/muta) with UDT token staking on CKB.
+Dapp developers could use Axon toolkit to create and distribute chain-specific UDT on CKB, chain token holders could staking UDT to become an Axon chain validator, or just deposit it into Axon chain for use in dapps.
+Axon will support [interoperability 2.0](https://medium.com/nervosnetwork/blockchain-abstraction-and-interoperability-2-0-eea98d81b7b6) and be 100% Ethereum compatible, like [Godwoken](https://github.com/nervosnetwork/godwoken).
+
+As one of the layer-2 solutions of [Nervos](https://www.nervos.org/), Axon could achieve thousands of TPS with the help of the high-performance consensus, [Overlord](https://github.com/nervosnetwork/overlord). 
+Thus just provides Nervos DApp developers with one more option. For low throughput high-value scenarios such as DeFi where security is usually a higher concern than speed, Godwoken is a perfect choice. 
+For high throughput consumer-facing applications such as decentralized games or social networks, Axon could play the best.
```
Comment (xying21) [inline README.md]: When it comes to consumer-facing applications such as decentralized games or social networks, Axon offers the best performance.

File: README.md [hunk 6]
```diff
@@ -1,5 +1,28 @@
...
+
+
+## Quick Start
+
+Clone the source code, locally compile and run a chain.
```
Comment (xying21) [inline README.md]: Clone the source code, compile locally, and run the chain.

File: README.md [hunk 7]
```diff
@@ -1,5 +1,28 @@
...
+```shell
+$ git clone https://github.com/nervosnetwork/axon.git
+$ cd axon
+$ cargo run --package axon --bin axon -- --config=./devtools/chain/config.toml --genesis=./devtools/chain/genesis_single_node.json
+```
+You can also refer to [axon-devops](https://github.com/nervosnetwork/axon-devops) to deploy multi-nodes chain.
```
Comment (xying21) [inline README.md]: You can also use [axon-devops](https://github.com/nervosnetwork/axon-devops) to deploy a multi-node chain.

File: README.md [hunk 8]
```diff
@@ -1,5 +1,28 @@
...
+
 ## Acknowledgment
 
 Axon is base on [Muta](https://github.com/nervosnetwork/muta). Sincerely thanks to everyone who [contributed code to Muta](https://github.com/nervosnetwork/muta/graphs/contributors).
```
Comment (xying21) [inline README.md]: Axon is based on [Muta](https://github.com/nervosnetwork/muta).

**Review by KaoImin — APPROVED** [review]

**Pull request merged**